### DISCUSSION TITLE
zasób customer-returns nie zwraca faktycznego nr przesyłki na etykiecie

### DISCUSSION THREAD

**Discussion by Zbigniew-L** [allegro/allegro-api#10068] [Suggestions - rejected]
Kiedy klient chce dokonać zwrotu allegro przyznaje dla tego zwrotu swój wewnętrzny numer. Następnie firma kurierska np. UPS generuje etykietę z numerem przesyłki, kiedy otrzymujemy dużo zwrotów chcielibyśmy skanować kod kreskowy na etykiecie kurierskiej i po tym skanie namierzyć zwrot kliencki - obecnie jest to możliwe ale nie dla wszystkich przewoźników (np. UPS) - ponieważ nr nr zwrotu w API to nie jest kod kreskowy z etykiety. 

Załączam przykładowe screeny zwrot1 - zwrot na allegro mamy referencje, etykieta UPS, referencja jest jako tekst, ale potrzebujemy w API uzyskać również/lub faktyczny kod kreskowy z etykiety

![zwrot1](https://github.com/user-attachments/assets/f131f56b-7549-445f-bdab-05c2404d18e7)
![zwrotUPS](https://github.com/user-attachments/assets/2b10bfe2-b1cf-4218-8d5e-03ef075d7238)
![zwrotAllegro](https://github.com/user-attachments/assets/98ea6100-557d-497b-9e31-d4ac93e9b6e3)

czyli w skrócie - byłoby idealnie gdyby waybill response zawierał faktycznie nr etykiety, pozdrawiam!

**Comment by AureliuszBrussy** [comment]
Dziękuję za sugestię. Jeśli zdecydujemy się na wprowadzenie takiej informacji, będziemy komunikować to na naszej stronie i repozytorium.

**Comment by devEcommercePL** [comment]
Również dołączam się do zgłoszenia

**Reply by AureliuszBrussy** [reply]
Dziękuję za Twój głos. Poinformujemy w tym wątku, jeżeli zdecydujemy się na takie wdrożenie.

**Comment by Lukasz-Zurek** [comment]
Zaktualizowaliśmy status sugestii jako "odrzucona", ponieważ nie planujemy sugerowanej zmiany.

Mamy wpływ na tzw. external reference, w którym wyświetla się numer zwrotu. Dbamy o to, by ów numer był widoczny na każdej etykiecie ponieważ po nim można zlokalizować zwrot. Jeśli natomiast chodzi o kod kreskowy/QR temat jest bardziej złożony, stąd nasza decyzja.
